bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinAirbaseUnitHandler.java

```diff
@@ -97,6 +97,16 @@ protected void pollStatus() throws IOException {
             updateTemperatureChannel(DaikinBindingConstants.CHANNEL_AC_TEMP, controlInfo.temp);
             updateState(DaikinBindingConstants.CHANNEL_AC_MODE, new StringType(controlInfo.mode.name()));
             updateState(DaikinBindingConstants.CHANNEL_AIRBASE_AC_FAN_SPEED, new StringType(controlInfo.fanSpeed.name()));
+
+            if (!controlInfo.power) {
+                updateState(DaikinBindingConstants.CHANNEL_AC_HOMEKITMODE, new StringType("off"));
```

Comment:
To make this consistent with the other channels, can you create an enum for "off","cool","heat" and "auto". It is easier to maintain channel options if they are backed by an enum in the codebase.

Reply:
I'll see what I can do over the weekend...

Reply:
This should be fixed in my latest commit